<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Курсовая работа</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="CameraGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>151</width>
      <height>351</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid gray;</string>
    </property>
    <property name="title">
     <string>Камера</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QGroupBox" name="CameraCoodinates">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>111</width>
       <height>151</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>10</pointsize>
       <stylestrategy>PreferAntialias</stylestrategy>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray;</string>
     </property>
     <property name="title">
      <string>Координаты</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <widget class="QLineEdit" name="CameraPosXEdit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>51</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>3000</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="CameraPosYEdit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>51</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="CameraPosZEdit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>110</y>
        <width>51</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>-1000</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>16</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 0px;

</string>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>16</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 0px;

</string>
      </property>
      <property name="text">
       <string>Y</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 0px;

</string>
      </property>
      <property name="text">
       <string>Z</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>111</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>Направление</string>
     </property>
     <widget class="QLineEdit" name="CameraDirXEdit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>51</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>-1</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="CameraDirYEdit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>51</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>16</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 0px;

</string>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>16</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 0px;

</string>
      </property>
      <property name="text">
       <string>Y</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QCheckBox" name="SmoothCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Сглаживание</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GeneratePicButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Генерировать 
изображение</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="BVHCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Использовать bvh 
дерево</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="MyPicture" name="picStandart" native="true">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid </string>
    </property>
   </widget>
   <widget class="MyPicture" name="picBVH" native="true">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>30</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Стандартная обратная трассировка лучей</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>10</y>
      <width>391</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Обратная трассировка лучей с использованием bvh дерева</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="DrawLightCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Рисовать 
источники света</string>
    </property>
   </widget>
   <widget class="QLabel" name="standartTime">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>540</y>
      <width>411</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bvhTime">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>540</y>
      <width>421</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Сцена</string>
    </property>
    <addaction name="EditProject"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="About"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="LoadProject"/>
    <addaction name="SaveProject"/>
    <addaction name="Exit"/>
   </widget>
   <addaction name="menu_3"/>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="LoadProject">
   <property name="text">
    <string>Загрузить сцену</string>
   </property>
  </action>
  <action name="SaveProject">
   <property name="text">
    <string>Сохранить сцену</string>
   </property>
  </action>
  <action name="EditProject">
   <property name="text">
    <string>Редактировать</string>
   </property>
  </action>
  <action name="About">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="Exit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyPicture</class>
   <extends>QWidget</extends>
   <header>mypicture.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
